recipes/b64/all/conandata.yml

```diff
@@ -0,0 +1,4 @@
+sources:
+  "1.2.1":
+    url: "https://github.com/KGrzeg/libb64-conan/archive/refs/tags/1.2.1_1.tar.gz"
```

Comment:
Please, use the original source from Source Forge https://sourceforge.net/projects/libb64/files/.

I see the upstream uses Makefile and VIsual Studio solution, which can be built with `Autotools` and `MSBuild` genereators/helpers.

The problem adding CMake is the maintenance, we avoid adding extra build helper files.

Reply:
@uilianries I have followed your advice and update my conan.py. The migration to autotools went smooth.
But there is a problem on Windows. Original sources contain visual studio solution, but the solution is not intended to build a library, but a CLI executable.
Is there any way to generate a fresh solution with conan's tools or I need to change the VS files on the fly with replace_in_file utility? (For now I have no idea what to change to obtain a desired effect though)

Reply:
@uilianries is it good now?